aalto/jupyterhub-instructors/faq-hints.rst

```diff
@@ -15,6 +15,10 @@ Instructions/hints
   answering questions about what to do with the data in the long
   term.  There can be a deputy who can also grant access.
 
+- To add more instructors/TAs, go to `domesti.cs.aalto.fi
+  <https://domesti.cs.aalto.fi>`_ and you can do it yourself.  You
+  must be connected to an Aalto network.
```

Comment:
Maybe add something like:
For connecting Aalto Network remotely please refer to [Aalto's VPN guide](
https://www.aalto.fi/en/services/establishing-a-remote-connection-vpn-to-an-aalto-network)

Reply:
thanks for the suggestion!